testapi.pm

```diff
@@ -475,14 +476,15 @@ subroutine block.
     send_key 'esc';
   }, 15);
 
-Returns true if screen changed or false on timeout. Default timeout is 10s. Default
-similarity_level is 50.
+Returns true if screen changed or false on timeout. Default timeout is 10s.
+Default minimal_timeout is 0.5s. Default similarity_level is 50.
 
 =cut
 
 sub wait_screen_change(&@) {
-    my ($callback, $timeout, %args) = @_;
-    $timeout ||= 10;
+    my ($callback, $timeout, $minimal_timeout, %args) = @_;
+    $timeout         ||= 10;
+    $minimal_timeout ||= 0.5;
```

Comment:
I would suggest to have a default timeout of 0, instead of 0.5, and add couple of tests in t/03-testapi.t

Reply:
Default 0 will be better, found out wait_screen_change is used also in another functions
Working on tests

Reply:
Mr. @dzedro Any updates here? :)

Reply:
Mr. @foursixnine no updates now, had no time to work on tests